spec/services/flags/destroy_flag_spec.rb

```diff
@@ -10,6 +10,8 @@
   let(:dependencies) { { guardian: current_user.guardian } }
   let(:flag_id) { flag.id }
 
+  after { flag.destroy }
```

Comment:
is it possible to use `destroy!` in all cases?

Reply:
Good catch, fixed!